### PR TITLE
Middle Mouse Close Option

### PR DESCRIPTION
**Pull request by fediaFedia**
Fantastic software! Can't believe I only now discovered it...
I do remember all the buzz around CairoShell back in 2007 hehe

Been using 7TaskbarTweaker before and it has this amazing feature to remap a middleclick to close a running app... kind of how you middle click to close Browser tabs.

I figured I'd build it for myself, but I saw an issue where someone also asked for it: https://github.com/dremin/RetroBar/issues/513

Might be a very popular and handy feature, hope this PR is good enough :)
By default, it's False

P.S. Went through the trouble to add the languages also...

rrDdmdyXtd

### CODE REVIEW

**Comment by xoascf** [comment]
Instead of just making it toggleable between two options, I think it would be better to make three selectable behavior options for the middle click button for tasks, something like:

- Close window
- Open new instance
- Do nothing

Or maybe this could be implemented for later...

Another thing to keep in mind is that this project uses spaces, not tabs, so it's preferable to change that to keep the code formatting consistent.

**Comment by xoascf** [comment]
I've seen some inconsistencies in the translations, I appreciate the work, but it's preferable to limit the PR to just adding this feature and adding the strings to the English version, if possible the translators will take care of updating them as they have been doing so far.

Thank you!

File: RetroBar/Languages/English.xaml [hunk 1]
```diff
@@ -83,5 +81,5 @@
 
     <FlowDirection x:Key="flow_direction">LeftToRight</FlowDirection>
 
-    <FontFamily x:Key="start_text_xp_font_family">Franklin Gothic</FontFamily>
```
Comment (xoascf) [inline RetroBar/Languages/English.xaml:84]: If you remove this, RetroBar may not work properly.

File: RetroBar/App.config [hunk 2]
```diff
@@ -37,7 +37,10 @@
             <setting name="ShowMultiMon" serializeAs="String">
                 <value>False</value>
             </setting>
-            <setting name="UseSoftwareRendering" serializeAs="String">
+			<setting name="UseSoftwareRendering" serializeAs="String">
+				<value>False</value>
+			</setting>
```
Comment (xoascf) [inline RetroBar/App.config:42]: For the sake of consistency in code formatting, it is preferable to use spaces instead of tabs.

Comment (xoascf) [reply RetroBar/App.config]: That was fast!

**Comment by fediaFedia** [comment]
Ok, good to know the translators can do this.
I removed all variations of it except for English.

I wouldn't do it as a "three-selectable" because the "Do Nothing" option seems like it wouldn't be very valuable to have, I think users can already do nothing by not middle clicking it. Besides, would be an extra click to set it up.

**Comment by fediaFedia** [comment]
> I've seen some inconsistencies in the translations, I appreciate the work, but it's preferable to limit the PR to just adding this feature and adding the strings to the English version, if possible the translators will take care of updating them as they have been doing so far.
> 
> Thank you!

Sorry about that, seems like I am unable to commit any whitespace changes

**Comment by xoascf** [comment]
The Do Nothing option was mainly because that was the default behavior in previous versions of Windows, the issue would be if the user does not want it to do nothing as they may press that button by mistake.

**Comment by xoascf** [comment]
Well, the only blank space left and those options maybe (or maybe not) we will add them later, for now this seems to be fine. Thanks!

**Review by xoascf — APPROVED** [review]

**Comment by fediaFedia** [comment]
Fair point, we are aiming for a trifecta of realism, utility and non-feature-creeping :)
For me personally just the middle clicking alone makes this utility so much more valuable, and I think anyone coming from 7TT will be pleasantly surprised to have it included :)

Thanks for your time

**Comment by dremin** [comment]
Thanks!

**Pull request merged**